### PR TITLE
updates through 17.2

### PR DESCRIPTION
**Pull request by speudusa**
Not ready for review.

### CODE REVIEW

**Comment by speudusa** [comment]
Ready for PR.  

added notes for using the repos and if learners started their own projects.

File: src/chapters/orm-intro/background.rst [hunk 1]
```diff
@@ -96,51 +102,136 @@ We now need to add a couple of NuGet packages to support our database connection
...
+
 Open the NuGet Package Manager in Visual Studio:
 
 - **Windows** - *Tools > NuGet Package Manager > Manage NuGet Packages for Solution*
 - **Mac** - *Project > Manage NuGet Dependencies*
 
-Search for ``Pomelo.EntityFrameworkCore.MySql``. Select the package and install. This dependency provides code that is able to connect to a MySQL database from within an ASP.NET Core application using EF. Note that this package itself depends on the main EntityFrameworkCore package, ``Microsoft.EntityFrameworkCore.Relational``, so it is also installed.
+Search for for all of the packages listed below. Select the package and install.
+Note the version numbers.  When using the package manager, you should be able to select a version.
+Match the versions provided below.
```
Comment (gildedgardenia) [inline src/chapters/orm-intro/background.rst:114]: Thinking about how we can avoid future confusion, what if you instead just specified that it needs to match the version of .NET they are using?

**Review by gildedgardenia — APPROVED** [review]
Just a minor suggestion for you!

File: src/chapters/orm-intro/background.rst (hunk 1 above)
Comment (speudusa) [reply src/chapters/orm-intro/background.rst:114]: Yes, we could totally do that.

**Pull request merged**